lesson05/assignment/index.html

```diff
@@ -1,15 +1,123 @@
+              </div>
+              <div class="main-container_articles_text">
+                <h4>Grab a warm drink and a tasty treat</h4>
+                <p>With exceptional espresso, pastries baked from scratch, gluten-free options, house-made granola, and delicious panini, Ten Ton Coffee has what you crave</p>
+              </div>
+            </article>
+            <article>
+              <div class="main-container_articles_wrapper">
+                <img src="img/ten-ton-mask.jpg" width="500" height="500">
+              </div>
+              <div class="main-container_articles_text">
+                <h4>We're open!</h4>
+                <p>Our fabulous take-out window is open daily, 7:30am - 2pm. Come grab a coffee and say hello!</p>
+              </div>
+            </article>
+          </div>
           
-        </main>
-    </body>
+          <section>
+            <h2>Instagram</h2> <!-- To be API feed -->
+            <div class="main-container_instagram">
+              <div class="main-container_instagram_wrapper">
+                <img src="img/ten-ton-insta-1.jpg" alt="Instagram image 1" width="250" height="250">
+              </div>
+              <div class="main-container_instagram_wrapper">
+                <img src="img/ten-ton-insta-2.jpg" alt="Instagram image 2" width="250" height="250">
+              </div>
+              <div class="main-container_instagram_wrapper">
+                <img src="img/ten-ton-insta-3.jpg" alt="Instagram image 3" width="250" height="250">
+              </div>
+              <div class="main-container_instagram_wrapper">
+                <img src="img/ten-ton-insta-4.jpg" alt="Instagram image 4" width="250" height="250">
+              </div>
+              <div class="main-container_instagram_wrapper">
+                <img src="img/ten-ton-insta-5.jpg" alt="Instagram image 5" width="250" height="250">
+              </div>
+              <div class="main-container_instagram_wrapper">
+                <img src="img/ten-ton-insta-1.jpg" alt="Instagram image 1" width="250" height="250">
+              </div>
+              <div class="main-container_instagram_wrapper">
+                <img src="img/ten-ton-insta-2.jpg" alt="Instagram image 2" width="250" height="250">
+              </div>
+              <div class="main-container_instagram_wrapper">
+                <img src="img/ten-ton-insta-3.jpg" alt="Instagram image 3" width="250" height="250">
+              </div>
+              <div class="main-container_instagram_wrapper">
+                <img src="img/ten-ton-insta-4.jpg" alt="Instagram image 4" width="250" height="250">
+              </div>
+              <div class="main-container_instagram_wrapper">
+                <img src="img/ten-ton-insta-5.jpg" alt="Instagram image 5" width="250" height="250">
+              </div>
+            </div>
+          </section>
+        </div>
+      </main>
+
+      <footer class="footer">
```

Comment:
Having a `footer` element removes the need to give it a class of "footer". Same for "header" and "main" elements. Each page should have only one, and there's no need for a redundant classname on them.